config/initializers/devise.rb

```diff
@@ -235,7 +235,7 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  config.omniauth :google_oauth2, ENV['DARIA_GOOGLE_KEY'], ENV['DARIA_GOOGLE_SECRET']
+  config.omniauth :google_oauth2, ENV['DARIA_GOOGLE_KEY'], ENV['DARIA_GOOGLE_SECRET'], {}
```

Comment:
PREGUNTA: What is the {} for?, new options you can pass that we aren't?

Reply:
I'm not sure if omniauth now takes a set number of arguments or if this is basically the same as what we were doing, but I followed the setup docs just to be on the safe side since we're bumping a bunch of major versions. Anyway, this is in there, figured it was best to just go with it.